examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,359 @@
+"""
+
+Modeling Cavitation
+-------------------
+This example examines the pressure-driven cavitating flow of water through a
+sharp-edged orifice. This is a typical configuration in fuel injectors, and
+brings a challenge to the physics and numerics of cavitation models because of
+the high pressure differentials involved and the high ratio of liquid to vapor
+density.
+
+This example uses the multiphase modeling capability of Ansys Fluent, and will
+be able to predict the strong cavitation near the orifice after flow separation
+at a sharp edge.
+
+**Workflow tasks**
+
+This example demonstrates how to do the following:
+
+- Set boundary conditions for internal flow
+- Use the mixture model with cavitation effects
+- Calculate a solution using the pressure-based coupled solver
+
+**Problem description**
+
+The problem considers the cavitation caused by the flow separation after a
+sharp-edged orifice. The flow is pressure driven, with an inlet pressure of 5 x
+10^5 Pa and an outlet pressure of 9.5 x 10^4 Pa. The orifice diameter is 4 x
+10^-3 m, and the geometrical parameters of the orifice are D/d = 2.88 and L/d =
+4, where D, d, and L are the inlet diameter, orifice diameter, and orifice
+length respectively.
+"""
+
+###############################################################################
+# Example Setup
+# -------------
+# Before you can begin, you must set up the example and initialize this
+# workflow.
+#
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports, which includes downloading and importing
+# the geometry file.
+
+# sphinx_gallery_thumbnail_path = '_static/cavitation_model.png'
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+cav_file = examples.download_file(
+    "cav.msh", "pyfluent/cavitation", save_path=pyfluent.EXAMPLES_PATH
+)
+
+###############################################################################
+# Launch Fluent
+# ~~~~~~~~~~~~~
+# Launch Fluent as a service in 2d solution mode with double precision running
+# on one processor.
+
+solver = pyfluent.launch_fluent(
+    precision="double",
+    processor_count=4,
+    mode="solver",
+    version="2d",
+    product_version="23.2.0",
```

Comment:
```suggestion
```
This specification is unnecessary for the documentation pipeline, the version is specified in the GitHub workflow file

Reply:
Yes, sorry forgot to check this.